vendor/github.com/Microsoft/go-winio/file.go

```diff
@@ -69,6 +69,7 @@ func initIo() {
 // win32File implements Reader, Writer, and Closer on a Win32 handle without blocking in a syscall.
 // It takes ownership of this handle and will close it if it is garbage collected.
 type win32File struct {
+	sync.Mutex
```

Comment:
There was actually 2 PR integrated in this one.

Sorry, I should have made it into a proper fork, I'll remember it next time, the vendor.conf does have v0.4.3 properly.